serverless.yml

```diff
@@ -1,3 +1,5 @@
+useDotenv: true
```

Comment:
This pulls in environment variables from the env files, choosing which env file to load depending on which environment we're in (`dev` vs `prod`)